app/bundles/EmailBundle/Config/config.php

```diff
@@ -90,6 +90,9 @@
             'mautic.emailbuilder.subscriber'         => array(
                 'class' => 'Mautic\EmailBundle\EventListener\BuilderSubscriber'
             ),
+            'mautic.emailtoken_subscriber'     => array(
```

Comment:
I know this is minor but wouldn't this be more uniform as `mautic.emailtoken.subscriber`?